GroceryStore/ProductDetailViewController.m

```diff
@@ -0,0 +1,123 @@
+//
+//  ProductDetailViewController.m
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "ProductDetailViewController.h"
+
+@interface ProductDetailViewController ()
+
+@property (weak, nonatomic) IBOutlet UILabel *productTitleLabel;
+@property (weak, nonatomic) IBOutlet UILabel *productQuantityLabel;
+
+@property (weak, nonatomic) IBOutlet UILabel *quantityAdjustmentLabel;
+@property (weak, nonatomic) IBOutlet UIStepper *stepperOutlet;
+
+@property (strong, nonatomic) NSMutableURLRequest *URLRequest;
+@property (strong, nonatomic) NSString *URLString;
+@property (strong, nonatomic) NSURLSession *session;
+
+@end
+@implementation ProductDetailViewController
+
+- (void)viewDidLoad {
+    
+    self.productTitleLabel.text = self.productName;
+    self.productQuantityLabel.text = [NSString stringWithFormat:@"Current quantity is: %@", self.quantity];
+}
+
+- (IBAction)stepped:(UIStepper *)sender {
+    
+    self.quantityAdjustmentLabel.text = [NSString stringWithFormat:@"%.0f", sender.value];
+    
+}
+
+- (IBAction)removeAllStock:(id)sender {
+    
+    //create session
+    [self createSessionWithURLComponent:@"inventory"];
```

Comment:
The session object is the same for all requests.  I'd suggest creating it once, independent of the action the user takes.
On a larger architectural note: consider combining all of the networking methods and logic into their own class.  That will make it easier to combine the repeated code (or strings or api paths) that are now spread across two VCs.  It will also make it more straightforward to swap NSURLSession to AFNetworking.